src/views/protected/case/Detail.vue

```diff
@@ -3,84 +3,150 @@
-        <div class="row m-0 justify-content-between align-items-end mt-3">
-            <span>
-                <img class="case-react-profile-pic rounded-circle" src="@/assets/profile-picture-1.jpg" alt="">
-                <img class="rel-right-1 case-react-profile-pic rounded-circle" src="@/assets/profile-picture-1.jpg" alt="">
-                <img class="rel-right-2 case-react-profile-pic rounded-circle" src="@/assets/profile-picture-1.jpg" alt="">
-            </span>
-            <small>10 proofs</small>
-        </div>
-
-        <!-- case comments -->
-        <hr class="mt-4 m-0">
-        <div class="pri-comments">
-            <List :caseUuid="1"></List>
-        </div>
+      <!-- case reaction box -->
+      <div class="reactions-box row justify-content-between m-0 mt-3">
+        <span class="row m-0 align-items-center">
+          <span class="row m-0 align-items-center">
+            <img
+              class="case-react-icons mr-1"
+              src="@/assets/case-up.svg"
+              alt=""
+            />
+            <small class="react-txt m-0 mr-3 h6">{{
+              caseDetail.metrics[0]
+            }}</small>
+          </span>
+          <span class="row m-0 align-items-center">
+            <img
+              class="case-react-icons mr-1"
+              src="@/assets/case-like.svg"
+              alt=""
+            />
+            <small class="react-txt m-0 mr-3 h6">{{
+              caseDetail.metrics[1]
+            }}</small>
+          </span>
+          <span class="row m-0 align-items-center">
+            <img
+              class="case-react-icons mr-1"
+              src="@/assets/case-dislike.svg"
+              alt=""
+            />
+            <small class="react-txt m-0 mr-3 h6">{{
+              caseDetail.metrics[2]
+            }}</small>
+          </span>
+        </span>
+        <span class="case-view-box row m-0 align-items-center">
+          <img class="case-view-icon" src="@/assets/case-view.svg" alt="" />
+          <small class="react-txt h6 m-0 pl-1">{{
+            caseDetail.metrics[3]
+          }}</small>
+        </span>
+      </div>
+      <!-- case reaction box 2 -->
+      <div class="row m-0 justify-content-between align-items-end mt-3">
+        <span>
+          <img
+            class="case-react-profile-pic rounded-circle"
+            src="@/assets/profile-picture-1.jpg"
+            alt=""
+          />
+          <img
+            class="rel-right-1 case-react-profile-pic rounded-circle"
+            src="@/assets/profile-picture-1.jpg"
+            alt=""
+          />
+          <img
+            class="rel-right-2 case-react-profile-pic rounded-circle"
+            src="@/assets/profile-picture-1.jpg"
+            alt=""
+          />
+        </span>
+        <small>10 proofs</small>
+      </div>
 
+      <!-- case comments -->
+      <hr class="mt-4 m-0" />
+      <div class="pri-comments">
+        <List :caseUuid="caseDetail.slug"></List>
```

Comment:
@pankajbedre : for your reference -> you need to convert this from `:caseUuid` to `:caseSlug`